setup.py

```diff
@@ -2,51 +2,46 @@
 
 setup(
     name="linnaeus",
-    version="v0.0.100",
+    version="v0.0.101",
```

Comment:
Is this a good time to declare this classifications v1.0.0? Seems like it might be